pkg/kubelet/container/runtime.go

```diff
@@ -114,6 +114,8 @@ type Runtime interface {
 	GetContainerLogs(ctx context.Context, pod *v1.Pod, containerID ContainerID, logOptions *v1.PodLogOptions, stdout, stderr io.Writer) (err error)
 	// Delete a container. If the container is still running, an error is returned.
 	DeleteContainer(containerID ContainerID) error
+	// Delete a sandbox. If the container is still running, an error is returned.
```

Comment:
linter would prefer to start the comment with the name of the method:
```suggestion
	// DeleteSandbox deletes a sandbox. If the container is still running, an error is returned.
```